tests/dag/config/base/dbt.yml

```diff
@@ -1,2 +1,5 @@
 target: dev
 target_type: bigquery
+vars:
+  variable_1: 123
+  variable_2: "var2"
```

Comment:
Unfortunately, it's not that easy to put here the Jinja templating — variable dictionary passed as `--vars` argument gets interpreted as a correct YAML and does not get processed by template engine. If one'd like support for templating here, it would require from `dp` to run template engine by itself (which can get even more problematic, as dbt uses [its own Jinja functions](https://docs.getdbt.com/reference/dbt-jinja-functions), e.g. `env_var`).